app/admin/study_code.rb

```diff
@@ -1,33 +1,32 @@
 ActiveAdmin.register StudyCode do
-    permit_params :title
+  permit_params :title
 
-    RESTRICTED_ACTIONS = ["new"]
+  RESTRICTED_ACTIONS = ["new"]
 
-
-    controller do
-        def action_methods
-            if StudyCode.count < 1
-                super
-            else
-                super - RESTRICTED_ACTIONS
-            end
-        end
+  controller do
+    def action_methods
+      if StudyCode.count < 1
+        super
+      else
+        super - RESTRICTED_ACTIONS
+      end
     end
+  end
 
-    index do
-        selectable_column
-        id_column
-        column :title
-        column :created_at
-        actions
-    end
+  index do
+    selectable_column
+    id_column
+    column :title
+    column :created_at
+    actions
+  end
 
-    filter :title
+  filter :title
 
-    form do |f|
-        f.inputs do
-            f.input :title
-        end
-        f.actions
+  form do |f|
+    f.inputs do
+      f.input :title, hint: hint(f.object.title)
```

Comment:
With the exception of this line, all the changes in this file are just formatting.